x-pack/legacy/plugins/siem/public/components/draggables/index.tsx

```diff
@@ -8,7 +8,6 @@ import { EuiBadge, EuiToolTip, IconType } from '@elastic/eui';
 import React from 'react';
 import styled from 'styled-components';
 
-import { Omit } from '../../../common/utility_types';
```

Comment:
I think (as per my IDE) this was the last ref to this export:

https://github.com/elastic/kibana/blob/27f5eaaaab35dca1c6c90acdf1cc96b10e2f6d5c/x-pack/plugins/siem/common/utility_types.ts#L13